rest/route/pod.go

```diff
@@ -80,3 +89,92 @@ func (h *podAgentCedarConfig) Run(ctx context.Context) gimlet.Responder {
 	}
 	return gimlet.NewJSONResponse(data)
 }
+
+////////////////////////////////////////////////
+//
+// PUT /rest/v2/pods/{pod_id}
```

Comment:
I think based on the way you wrote it, it makes more sense to change this to a POST since it's only going to do an insert and is non-idempotent, so I would change this to `POST /rest/v2/pods`.

We won't need pod ID since it's already randomized in `(*APICreatePod).ToService`.